Lombiq.Tests.UI/Services/OrchardCoreInstance.cs

```diff
@@ -53,6 +55,7 @@ public sealed class OrchardCoreInstance : IWebApplicationInstance
     private static readonly PortLeaseManager _portLeaseManager;
     private static readonly ConcurrentDictionary<string, string> _exeCopyMarkers = new();
     private static readonly object _exeCopyLock = new();
+    private static readonly bool _isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
Now, I've noticed it: We use `OperatingSystem.IsWindows()` in `Lombiq.HelpfulLibraries.Cli.Helpers.OperatingSystemHelper.GetExecutableExtension()`, but here we have this. Any specific reason for this choice?

~Googling~ Ducking this led me to the following interesting comment under [this answer](https://stackoverflow.com/a/38790956/177710):
![image](https://user-images.githubusercontent.com/164360/170370765-631e9bdb-bfef-4bef-a233-171a2dda8d68.png)

Does it have to do with that?

Reply:
I think the term you are looking for is... "searching"? 😁 

I can't think of a particular reason, and `OperatingSystem.IsWindows()` uses the build target to determine the OS as well:
![image](https://user-images.githubusercontent.com/4007293/170374717-d9b27752-dd7c-4013-bf8d-7dc420fa8120.png)

And IsOSPlatform:
![image](https://user-images.githubusercontent.com/4007293/170375090-929495c8-49d7-4f30-801c-0a59ede4a8d0.png)

Since both use preprocessor directives there is no functionality or performance difference. At runtime both are simply constant-returning static methods.

I've changed `_isWindows ` to use `OperatingSystem.IsWindows()` for simplicity.

Reply:
I have to become bolder at simply looking at the source, too!